cmd/vela-s3-cache/rebuild.go

```diff
@@ -111,13 +121,21 @@ func (r *Rebuild) Validate() error {
 
 	// verify timeout is provided
 	if strings.EqualFold(r.Timeout.String(), "0s") {
```

Comment:
Can we simplify this?

I think this could be modified to be:

```suggestion
	if r.Timeout == 0 {
```